docs/schema-loader-import.md

```diff
@@ -0,0 +1,271 @@
+| blob          | BLOB            | [*3](#warn-data-size) |
+| char          | TEXT            | [*2](#warn-data-size) |
+| clob          | TEXT            |                       |
+| float         | DOUBLE          | [*4](#warn-data-size) |
+| long          | TEXT            |                       |
+| long raw      | BLOB            |                       |
+| nchar         | TEXT            | [*2](#warn-data-size) |
+| nclob         | TEXT            |                       |
+| number        | BIGINT / DOUBLE | [*5](#warn-data-size) |
+| nvarchar2     | TEXT            | [*2](#warn-data-size) |
+| raw           | BLOB            | [*2](#warn-data-size) |
+| varchar2      | TEXT            | [*2](#warn-data-size) |
+
+Data types not listed in the above are not supported. The typical examples are shown below.
+
+- date
+- timestamp
+- interval
+- rowid
+- urowid
+- bfile
+- json
+
+</div>
+
+<div id="SQLServer" class="tabcontent" markdown="1">
+
+| SQL Server | ScalarDB | Notes                 |
+|------------|----------|-----------------------|
+| bigint     | BIGINT   | [*1](#warn-data-size) |
+| binary     | BLOB     | [*2](#warn-data-size) |
+| bit        | BOOLEAN  |                       |
+| char       | TEXT     | [*2](#warn-data-size) |
+| float      | DOUBLE   |                       |
+| image      | BLOB     |                       |
+| int        | INT      |                       |
+| nchar      | TEXT     | [*2](#warn-data-size) |
+| ntext      | TEXT     |                       |
+| nvarchar   | TEXT     | [*2](#warn-data-size) |
+| real       | FLOAT    |                       |
+| smallint   | INT      | [*2](#warn-data-size) |
+| text       | TEXT     |                       |
+| tinyint    | INT      | [*2](#warn-data-size) |
+| varbinary  | BLOB     | [*2](#warn-data-size) |
+| varchar    | TEXT     | [*2](#warn-data-size) |
+
+Data types not listed in the above are not supported. The typical examples are shown below.
+
+- cursor
+- date
+- datetime
+- datetime2
+- datetimeoffset
+- decimal
+- geography
+- geometry
+- hierarchyid
+- money
+- numeric
+- rowversion
+- smalldatetime
+- smallmoney
+- sql_variant
+- time
+- uniqueidentifier
+- xml
+
+</div>
+
+</div>
+
+{% capture notice--warning %}
+**Attention**
+
+1. The value range of `BIGINT` in ScalarDB is from -2^53 to 2^53, regardless of the size of `bigint` in the underlying database. Thus, if the data out of this range exists in the imported table, ScalarDB cannot read it.
+2. For certain data types noted above, ScalarDB may map a data type larger than that of the underlying database. In that case, You will see errors when putting a value with a size larger than the size specified in the underlying database.
+3. The maximum size of `BLOB` in ScalarDB is about 2GB (precisely 2^31-1 bytes). In contrast, Oracle `blob` can have (4GB-1)*(number of blocks). Thus, if the data larger than 2GB exists in the imported table, ScalarDB cannot read it.
```

Comment:
Ditto. Do we observe `null`, some value, or an error?

Reply:
Good question again. The 2GB limit is due to Java's byte array limits. I don't test it, but I guess the Oracle JDBC driver throws an `SQLException`. Or, we might see an OOM error if the heap size is not correctly configured. Apart from that, we might be able to handle such large objects better by using JDBC `Blob getBlob(...)` and offset-based access instead of `byte[] getBytes(...)`.

Reply:
Ah, I see. I understood that it depends on Java side.
Thank you!